src/qunit.cpp

```diff
@@ -1098,8 +1120,8 @@ bool QUnit::TryCnotOptimize(const bitLenInt* controls, const bitLenInt& controlL
         ApplySingleInvert(topRight, bottomLeft, true, target);
         return true;
     } else if (rControlLen == 1U) {
-        complex iTest[4] = { bottomLeft, 0, 0, topRight };
-        if (IsIdentity(iTest)) {
+        complex iTest[4] = { topRight, 0, 0, bottomLeft };
```

Comment:
Is this code path covered by a unit test now?

Reply:
It was, and still is.